cmd/account/rotate-secret.go

```diff
@@ -34,38 +32,17 @@ func newCmdRotateSecret(streams genericclioptions.IOStreams, flags *genericcliop
 		},
 	}
 
-	rotateSecretCmd.Flags().StringVar(&ops.accountNamespace, "account-namespace", common.AWSAccountNamespace,
-		"The namespace to keep AWS accounts. The default value is aws-account-operator.")
-
-	rotateSecretCmd.Flags().StringVarP(&ops.accountName, "account-name", "a", "", "AWS Account CR name")
-	rotateSecretCmd.Flags().StringVarP(&ops.accountID, "account-id", "i", "", "AWS Account ID")
 	rotateSecretCmd.Flags().StringVarP(&ops.profile, "aws-profile", "p", "", "specify AWS profile")
-	rotateSecretCmd.Flags().StringVarP(&ops.cfgFile, "aws-config", "c", "", "specify AWS config file path")
-	rotateSecretCmd.Flags().StringVarP(&ops.region, "aws-region", "r", common.DefaultRegion, "Specify AWS region")
-	rotateSecretCmd.Flags().StringVar(&ops.secretName, "secret-name", "byoc", "Specify name of the generated secret")
-	rotateSecretCmd.Flags().StringVar(&ops.secretNamespace, "secret-namespace", "aws-account-operator", "Specify namespace of the generated secret")
-	rotateSecretCmd.Flags().BoolVar(&ops.printSecret, "print", true, "Print the generated secret")
-	rotateSecretCmd.Flags().StringVarP(&ops.outputPath, "output", "o", "", "Output path for secret yaml file")
+	rotateSecretCmd.Flags().BoolVar(&ops.osdCcsAdmin, "ccs", false, "Also rotates osdCcsAdmin credential. Use caution.")
```

Comment:
I suggest we rename that variable to `ops.updateCcsCreds` or something that describes the action

Reply:
👍